ipatests/test_webui/test_cert.py

```diff
@@ -162,16 +164,10 @@ def test_search_revocation_reason(self):
         self.init_app()
         self.navigate_to_entity(ENTITY)
 
-        # add a new cert
-        hostname = self.config.get('ipa_server')
-        csr = generate_csr(hostname)
-        record = add_cert(self, 'HTTP/{}'.format(hostname), csr)
-
-        # revoke added cert
-        revoke_cert(self, record, '1')
+        # revoke new certificate
+        self._add_and_revoke_cert()
 
         # search cert by revocation reason
-        self.navigate_to_entity(ENTITY)
```

Comment:
Why this needs to be removed, while similar calls before revoking the certificates were kept intact?

Reply:
Yes, calls before should be also moved to the function. I'll fix it.

Reply:
@netoarmando 
Fixed, tests have passed.